Guided Investigation - MDATP Webshell Alerts.ipynb

```diff
@@ -0,0 +1,841 @@
+{
```

Comment:
<p>You could use the nbwidgets.QueryTime here but you'd have to change the query below - next time maybe.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/28/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Top tip, will use next time</p>